cfgov/unprocessed/apps/mmt-cost-of-credit-tool/js/components/button.js

```diff
@@ -0,0 +1,54 @@
+import clsx from 'clsx';
```

Comment:
This is a duplicate of a module that exists in My Money Calendar. Ultimately we should find a sensible way for both tools to use the same button module, since this is a React implementation of CFPB's design system button component that we can use in an unlimited number of CFPB react apps.

Reply:
I'd like to leave it for now.  We can talk and figure out how to handle shared components at some point.